__tests__/endpoints.test.js

```diff
@@ -74,6 +74,22 @@ describe('NC news endpoint tests',()=>{
                     expect(body.msg).toBe("article does not exist")
                 })
             })
+            test('Respond with 200 with the correct article and its comment count',()=>{
+                return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then(({body:{article}})=>{
+                    expect(article.article_id).toBe(1);
+                    expect(typeof article.topic).toBe('string');
+                    expect(typeof article.topic).toBe('string');
+                    expect(typeof article.author).toBe('string');
+                    expect(typeof article.body).toBe('string');
+                    expect(typeof article.created_at).toBe('string');
+                    expect(typeof article.votes).toBe('number');
+                    expect(typeof article.article_img_url).toBe('string');
+                    expect(typeof article.comment_count).toBe('number')
```

Comment:
I think this could check for an exact value, as we can determine how many comments are associated with with article via the test-data